test/benchmarks/bench_classify.rb

```diff
@@ -38,7 +38,7 @@ def setup
   end
 
   def bench_allocations
-    assert_allocations "3.0" => 6, "2.7" => 4, "2.6" => 4, "2.5" => 4 do
+    assert_allocations "3.1" => 6, "3.0" => 6, "2.7" => 4, "2.6" => 4 do
```

Comment:
I arbitrary set it to 6.

Reply:
That's probably right. I would just set it to the number the test failure spits out, if any.